### PR TITLE
Create transfer complete page.

### PR DESCRIPTION
**Pull request by MancunianSam**
We already had a placeholder but this is implementing the design for the page.
We're not displaying a specific error if triggering the export fails so I've removed the parameter and the test for it.
I've also renamed everything from transfer confirmation to transfer complete to match with the title.
I've also changed the export graphql and export api calls to fail if there are errors rather than recovering from them and returning false as they're doing at the moment.

### CODE REVIEW

File: test/controllers/TransferCompleteControllerSpec.scala [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import java.util.UUID
+
+import play.api.test.CSRFTokenHelper.CSRFRequest
+import play.api.test.FakeRequest
+import play.api.test.Helpers.{GET, contentAsString, defaultAwaitTimeout}
+import util.FrontEndTestHelper
+
+class TransferCompleteControllerSpec extends FrontEndTestHelper {
+
+  "TransferConfirmationController GET" should {
```
Comment (TomJKing) [inline test/controllers/TransferCompleteControllerSpec.scala:12]: Should this read: "Transfer**Complete**Controller ..."

File: test/controllers/TransferSummaryControllerSpec.scala [hunk 2]
```diff
@@ -155,6 +155,7 @@ class TransferSummaryControllerSpec extends FrontEndTestHelper {
     "redirects to the transfer confirmation page when a valid form is submitted" in {
```
Comment (TomJKing) [inline test/controllers/TransferSummaryControllerSpec.scala:155]: Should this read: "... to the transfer **complete** page .."?

**Review by TomJKing — COMMENTED** [review]
Looks good. Couple of renaming comments.

File: test/controllers/TransferCompleteControllerSpec.scala (hunk 1 above)
Comment (MancunianSam) [reply test/controllers/TransferCompleteControllerSpec.scala]: It should yeah, cheers. I've changed it.

File: test/controllers/TransferSummaryControllerSpec.scala (hunk 2 above)
Comment (MancunianSam) [reply test/controllers/TransferSummaryControllerSpec.scala]: I've changed this one as well.

**Review by TomJKing — APPROVED** [review]

**Pull request merged**